client/app/kubernetes/nodes/kubernetes-nodes.controller.js

```diff
@@ -56,15 +56,16 @@ angular.module('managerApp').controller('KubernetesNodesCtrl', class KubernetesN
   }
 
   getAssociatedFlavor(node) {
-    if (node.instanceId) {
-      return this.Kubernetes.getAssociatedInstance(node.projectId, node.instanceId)
```

Comment:
Is `getAssociatedInstance` method from the `Kubernetes` still need? 

```sh
$ git grep -i 'getAssociatedInstance'
client/app/kubernetes/kubernetes.service.js:  getAssociatedInstance(projectId, instanceId) {
client/app/kubernetes/nodes/kubernetes-nodes.controller.js:      return this.Kubernetes.getAssociatedInstance(node.projectId, node.instanceId)
```

Reply:
This method is no longer being used anywhere. Hence it has been removed, Thanks for pointing it out :)